analyticsdataserver/settings/base.py

```diff
@@ -202,6 +202,7 @@
 )
 
 THIRD_PARTY_APPS = (
+    'release_util',
```

Comment:
Nit: I'm not sure that it matters practically, but I think that the import order should be consistent between https://github.com/edx/edx-analytics-dashboard/pull/500 and this.  My slight preference is for it to go over the `rest_framework` imports.